test/Microsoft.AspNet.Mvc.FunctionalTests/ModelBindingFromQueryTest.cs

```diff
@@ -0,0 +1,143 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Builder;
+using Microsoft.AspNet.TestHost;
+using ModelBindingWebSite;
+using Newtonsoft.Json;
+using Xunit;
+using ModelBindingWebSite.Controllers;
+
+namespace Microsoft.AspNet.Mvc.FunctionalTests
+{
+    public class FromQueryAttributeTest
```

Comment:
naming nits and so on from `ModelBindingFromFormTest` apply to other new functional test classes